src/main/java/lotto/domain/WinningNumber.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain;
+
+public final class WinningNumber {
+
+    private final Lotto winningLotto;
+    private final LottoNumber bonusBall;
+
+    public WinningNumber(final String lottoValue, final String bonusValue) {
+        TextParser lottoText = new TextParser(lottoValue);
+        Lotto lotto = new Lotto(lottoText.getParsed());
+        LottoNumber lottoNumber = new LottoNumber(bonusValue);
+
+        validate(lotto, lottoNumber);
+
+        winningLotto = lotto;
+        bonusBall = lottoNumber;
+    }
+
+    private void validate(final Lotto lotto, final LottoNumber lottoNumber) {
+        if (lotto.hasLottoNumber(lottoNumber)) {
+            throw new IllegalArgumentException("보너스볼이 당첨번호에 존재합니다");
```

Comment:
에러 메시지를 작성할 때 어떤 사유로 발생했는지 작성해주면 두고두고 큰 도움을 받을 수 있습니다
```suggestion
            throw new IllegalArgumentException(String.format("보너스볼이 당첨번호에 존재합니다. 보너스볼 번호 : %d", LottoNumber.getNumber));
```